src/VirtualClient/VirtualClient.Actions.FunctionalTests/SysbenchOLTP/SysbenchOLTPProfileTests.cs

```diff
@@ -0,0 +1,666 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using VirtualClient.Common;
+    using VirtualClient.Contracts;
+
+    [TestFixture]
+    [Category("Functional")]
+    public class SysbenchOLTPProfileTests
```

Comment:
What's the difference between this profile test and the ClientProfileTests? Looks like you are validating the same profile?